support/cas-server-support-sms-smsmode/src/test/java/org/apereo/cas/support/sms/SmsModeSmsSenderTests.java

```diff
@@ -2,20 +2,23 @@
 
 import org.apereo.cas.config.SmsModeSmsConfiguration;
 import org.apereo.cas.notifications.sms.SmsSender;
-import org.apereo.cas.util.MockWebServer;
 
+import com.sun.net.httpserver.HttpExchange;
```

Comment:
Are you sure this is the HTTP server you should use, part of the `com.sun` package? Is this not an internal JDK class?

Reply:
Yup, looks like it is.

Reply:
Indeed, the internal JDK package are `sun.*` (not `com.sun`).